kedro/framework/cli/cli.py

```diff
@@ -85,10 +85,13 @@ def info():
 
 @cli.command(short_help="See the kedro API docs and introductory tutorial.")
 def docs():
-    """Display the API docs and introductory tutorial in the browser,
-    using the packaged HTML doc files."""
-    html_path = str((Path(__file__).parent.parent / "html" / "index.html").resolve())
-    index_path = f"file://{html_path}"
+    """Display the online API docs and introductory tutorial in the browser."""
```

Comment:
```suggestion
    """Display the online API docs and introductory tutorial in the browser. (DEPRECATED)"""
```
Just for consistency with how we deprecated commands before.